app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/autoAnalysisTab.jsx

```diff
@@ -1,125 +1,52 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import { injectIntl, defineMessages, intlShape } from 'react-intl';
-import { InputBigSwitcher } from 'components/inputs/inputBigSwitcher';
-import { InputRadio } from 'components/inputs/inputRadio';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { projectAnalyzerConfigSelector, fetchProjectAction } from 'controllers/project';
+import { activeProjectSelector } from 'controllers/user';
+import { StrategyBlock } from './strategyBlock';
+import { IndexActionsBlock } from './indexActionsBlock';
 import styles from './autoAnalysisTab.scss';
 
 const cx = classNames.bind(styles);
 
-const messages = defineMessages({
-  autoAnalysisSwitcherTitle: {
-    id: 'AutoAnalysisSwitcher.title',
-    defaultMessage: 'Auto analysis',
+@connect(
+  (state) => ({
+    analyzerConfiguration: projectAnalyzerConfigSelector(state),
```

Comment:
Why do you connect analyzerConfiguration here? It will be better to do it in StrategyBlock component